-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 10, Failures: 8, Errors: 0, Skipped: 0, Time elapsed: 13 sec <<< FAILURE!
scenario(dev.serverest.runners.TestNGRunner)  Time elapsed: 4.485 sec  <<< FAILURE!
io.cucumber.testng.UndefinedStepException: The step 'o status code será 201' is undefined.
You can implement this step using the snippet(s) below:
@Entao("o status code será {int}")
public void oStatusCodeSerá(Integer int1) {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}

	at io.cucumber.core.runtime.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:69)
	at io.cucumber.testng.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:23)
	at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:133)
	at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
	at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
	at dev.serverest.runners.TestNGRunner.scenario(TestNGRunner.java:31)

scenario(dev.serverest.runners.TestNGRunner)  Time elapsed: 0.505 sec  <<< FAILURE!
io.cucumber.testng.UndefinedStepException: The step 'o status code será 201' is undefined.
You can implement this step using the snippet(s) below:
@Entao("o status code será {int}")
public void oStatusCodeSerá(Integer int1) {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}

	at io.cucumber.core.runtime.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:69)
	at io.cucumber.testng.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:23)
	at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:133)
	at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
	at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
	at dev.serverest.runners.TestNGRunner.scenario(TestNGRunner.java:31)

scenario(dev.serverest.runners.TestNGRunner)  Time elapsed: 0.003 sec  <<< FAILURE!
io.cucumber.testng.UndefinedStepException: The step 'tenho uma conta já cadastrada' and 1 other step(s) are undefined.
You can implement these steps using the snippet(s) below:
@Dado("tenho uma conta já cadastrada")
public void tenhoUmaContaJáCadastrada() {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}
@Entao("o status code será {int}")
public void oStatusCodeSerá(Integer int1) {
    // Write code here that turns the phrase above into concrete actions
    throw new io.cucumber.java.PendingException();
}

	at io.cucumber.core.runtime.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:69)
	at io.cucumber.testng.TestCaseResultObserver.assertTestCasePassed(TestCaseResultObserver.java:23)
	at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$1(TestNGCucumberRunner.java:133)
	at io.cucumber.core.runtime.CucumberExecutionContext.lambda$runTestCase$5(CucumberExecutionContext.java:137)
	at io.cucumber.core.runtime.RethrowingThrowableCollector.executeAndThrow(RethrowingThrowableCollector.java:23)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:137)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:129)
	at dev.serverest.runners.TestNGRunner.scenario(TestNGRunner.java:31)

Login(dev.serverest.tests.LoginTest)  Time elapsed: 0.35 sec  <<< FAILURE!
java.lang.AssertionError: expected [200] but found [401]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1419)
	at org.testng.Assert.assertEquals(Assert.java:1383)
	at org.testng.Assert.assertEquals(Assert.java:1429)
	at dev.serverest.tests.LoginTest.Login(LoginTest.java:57)

ProdutosGet(dev.serverest.tests.LoginTest)  Time elapsed: 0.423 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.fail(Assert.java:116)
	at dev.serverest.api.applicationAPI.ProdutosAPI.assertProductNameThroughList(ProdutosAPI.java:49)
	at dev.serverest.tests.LoginTest.ProdutosGet(LoginTest.java:68)

UsuariosDelete(dev.serverest.tests.LoginTest)  Time elapsed: 0.315 sec  <<< FAILURE!
java.lang.AssertionError: expected [Registro excluÃ­do com sucesso] but found [Nenhum registro excluído]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at dev.serverest.tests.LoginTest.UsuariosDelete(LoginTest.java:40)

UsuariosGet(dev.serverest.tests.LoginTest)  Time elapsed: 0.322 sec  <<< FAILURE!
java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.fail(Assert.java:116)
	at dev.serverest.api.applicationAPI.UsuariosAPI.assertNameThroughUserList(UsuariosAPI.java:78)
	at dev.serverest.tests.LoginTest.UsuariosGet(LoginTest.java:32)

UsuariosPut(dev.serverest.tests.LoginTest)  Time elapsed: 0.413 sec  <<< FAILURE!
java.lang.AssertionError: expected [Registro alterado com sucesso] but found [Este email já está sendo usado]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at dev.serverest.tests.LoginTest.UsuariosPut(LoginTest.java:50)

